plugin_build/wrap.go.tmpl

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
I suggest making these files `.go` files again, but with a `+build ignore` so that:

* they don't break the regular build
* tools like `gofmt` and `goimports` still fix them

Reply:
There's an example on how to do it here: https://golang.org/pkg/go/build/#hdr-Build_Constraints